tests/src/test/java/alluxio/testutils/LocalAlluxioClusterResource.java

```diff
@@ -164,8 +167,19 @@ public void evaluate() throws Throwable {
             .getTestName(description.getTestClass().getSimpleName(), description.getMethodName());
         try {
           try {
+            Annotation configAnnotation;
+            configAnnotation = description.getAnnotation(ServerConfig.class);
+            if (configAnnotation != null) {
+              ServerConfig config = (ServerConfig) configAnnotation;
+              // Override the configuration parameters with any configuration params
+              for (int i = 0; i < config.confParams().length; i += 2) {
+                mConfiguration.put(PropertyKey.fromString(config.confParams()[i]),
+                    config.confParams()[i + 1]);
+              }
+            }
```

Comment:
It seems like this could share code with the block below as well...

Reply:
I slightly refactored it, but there are still some similarities.